### ISSUE TITLE
Serve documentation statically

### ISSUE THREAD

**Issue by antisaling** [opendatahub-io/model-registry-bf4-kf#120]
Documentation should be built by CI and served statically. This helps us reference it in other parts of the code, as well as providing accessibility to users.
